### ISSUE TITLE
Pandas dataframes with NaN values

### ISSUE THREAD

**Issue by eliasmistler** [ideonate/nb2xls#3]
# Issue Description
The export function fails when there are pandas dataframes with NaN values. 

The following message is displayed:
```
nbconvert failed: NAN/INF not supported in write_number() without 'nan_inf_to_errors' Workbook() option
```

# Reproducing
To reproduce the issue, simply create a notebook with the following cell and try to export:
```python
pd.DataFrame([[pd.np.nan]])
```

**Comment by danlester** [comment]
Thanks a lot for reporting. I should get a chance to look at this in the next couple of days.

**Comment by danlester** [comment]
I've just released version 0.1.3 to deal with nans better - it should show as NA() in Excel now. I really appreciate your feedback - thanks again for letting me know!
